Sources/o/File/o+file.swift

```diff
@@ -142,10 +163,64 @@ public extension o.file {
 
         var data = try JSONEncoder().encode(value)
         
+        try out(
+            data: data,
+            path: path,
+            filename: filename,
+            base64Encoded: base64Encoded
+        )
+    }
+
+    /// Write a string to a file
+    ///
+    /// - Parameters:
+    ///   - string: The string to write to the file.
+    ///   - path: The path to the directory where the file should be written. The default is `.`, which means the current working directory.
+    ///   - filename: The name of the file to write.
+    ///   - using: The String.Encoding to encode the string with. The default is `.utf8`.
+    ///   - base64Encoded: A Boolean value indicating whether the data should be Base64-encoded before writing to the file. The default is `true`.
+    /// - Throws: If there's an error writing the data to the file.
+    static func out(
+        string: String,
+        path: String = ".",
+        filename: String,
+        using stringEncoding: String.Encoding = .utf8,
+        base64Encoded: Bool = true
+    ) throws {
+        let directoryURL = URL(fileURLWithPath: path)
```

Comment:
`fileURLWithPath` has a deprecation warning in Xcode 14.2

Reply:
Added a private helper to handle both versions for now. https://github.com/0xOpenBytes/o/pull/7/commits/55dd8504a818b1be71f5c0edcf817ef2e77e6e77